app/src/main/java/com/amaze/filemanager/asynchronous/services/ftp/FtpService.java

```diff
@@ -112,6 +123,10 @@
 
     private boolean isStartedByTile = false;
 
+    private static NotificationCompat.Builder builder;
```

Comment:
Why is this static? Shouldn't it get destroyed when the service is destroyed and recreated when needed?